dali/ft/filecopy.cpp

```diff
@@ -1522,6 +1522,9 @@ IAPICopyClient * FileSprayer::getAPICopyClient()
     if (needCalcCRC && !sources.item(0).hasCRC)
         return nullptr;
 
+    if (!distributedSource)
+        return nullptr;
+
```

Comment:
I think this change is good.  There should also be checks that sourcePlane and targetPlane are not null before they are derefenced.

Reply:
Two more checks added